### DISCUSSION TITLE
How to get the false subcondition?

### DISCUSSION THREAD

**Discussion by xidian237** [antonmedv/expr#509] [Q&A] [answered]
Hi experts,
I wonder if it's possible to get the false sub condition, while the expression consists of multi conditions which are divided by "()","&&" and "||", such as "(x > 0 && y < 10) || z == 'expr'". If the "x>0" is false, I want to return the false condition "x>0".

**Comment by antonmedv** [comment]
What about this?

    x > 0 ? (y < 10 || z == “expr”) : false

**Reply by xidian237** [reply]
@antonmedv , thank you for your comment, but you might misunderstand my question.
For example, now I have the expression: 
expression:="(condition1 && condition2) || (condition3 && condition4) || condition5", 
and the result of expr.eval(expression) is false, so I want to find which condition causes the whole expression to be false.

**Reply by antonmedv** [reply, marked as answer]
I see. For this we need to implement something like “explainer” #200 

I actually like this idea and will try to figure out something to solve it.

**Reply by xidian237** [reply]
@antonmedv , Thank you~
